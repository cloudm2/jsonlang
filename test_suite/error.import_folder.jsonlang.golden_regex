(STATIC ERROR: lib:1:1: Unexpected end of file.|RUNTIME ERROR: Couldn't open import "lib": basic_filebuf::underflow error reading the file(: iostream error)?
	error.import_folder.jsonlang:17:1-12	)
