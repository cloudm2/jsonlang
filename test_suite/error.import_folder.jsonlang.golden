RUNTIME ERROR: Couldn't open import "lib": basic_filebuf::underflow error reading the file
	error.import_folder.jsonlang:17:1-12	
